### ISSUE TITLE
Fix file-manager!

### ISSUE THREAD

**Issue by LsMacox** [linuxdeepin/dde-file-manager#273]
![image](https://user-images.githubusercontent.com/52324022/85197718-7ea4a700-b2eb-11ea-9515-31b14bf0f231.png)
Plese fix

**Comment by mitchellkrogza** [comment]
I agree, the latest version is so full of bugs and not been tested at all

**Comment by LsMacox** [comment]
> I agree, the latest version is so full of bugs and not been tested at all

no support at all

**Comment by Vistaus** [comment]
Can't reproduce this.
